merchant-app/app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".BillingActivity"></activity>
```

Comment:
Why not close the tag like the next line ?

Reply:
sure